pkg/constants/constants.go

```diff
@@ -73,6 +73,14 @@ const (
 	AgentModelDirArgName  = "--model-dir"
 )
 
+// InferenceGraph Annotations
+var (
+	InferenceGraphMinScaleAnnotationKey = KServeAPIGroupName + "/min-scale"
+	InferenceGraphMaxScaleAnnotationKey = KServeAPIGroupName + "/max-scale"
+	InferenceGraphTargetAnnotationKey   = KServeAPIGroupName + "/target"
+	InferenceGraphMetricsAnnotationKey  = KServeAPIGroupName + "/metrics"
```

Comment:
We should be consistent with `InferenceService` using properly defined fields on spec instead of annotations.

Reply:
Agreed. Implemented accordintly